src/main/java/seedu/address/model/module/ModuleLesson.java

```diff
@@ -0,0 +1,60 @@
+package seedu.address.model.module;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a lesson of a module.
+ * Guarantees: immutable; is valid as declared in {@link #isValidLesson(String)}
+ */
```

Comment:
I think we should also explain that this class only represents the name of a lesson of a module? Since it can be misinterpreted to store the whole information of the lesson ?

Reply:
Yes, I think we should. Thanks for the note!